tasks/appraisal.rake

```diff
@@ -0,0 +1,187 @@
+# frozen_string_literal: true
+
+namespace :appraisal do # rubocop:disable Metrics/BlockLength
```

Comment:
I pondered about that.

I opted for this as it's meant to be a drop-in-ish† replacement for the upstream tasks that got removed.

Maybe a `:dd` toplevel namespace would be in order but I feel like it should be part of a larger refactoring.

† -ish because it's not technically doing *exactly* the same thing but it's close enough in intent and CLI interface

Reply:
Yeah, I somewhat fear that someone may be confused on where these tasks are coming from, and have to spend some time looking at upstream before realizing they're ours, placed inside upstream's namespace.